{
    "Information": {
        "Category": null, 
        "CAPEC": null, 
        "CVE": [
            {
                "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2013-2099", 
                "published": "2013-10-09T10:53:20.230-04:00", 
                "id": "CVE-2013-2099", 
                "modified": "2014-12-02T21:59:49.497-05:00", 
                "summary": "Algorithmic complexity vulnerability in the ssl.match_hostname function in Python 3.2.x, 3.3.x, and earlier, and unspecified versions of python-backports-ssl_match_hostname as used for older Python versions, allows remote attackers to cause a denial of service (CPU consumption) via multiple wildcard characters in the common name in a certificate."
            }
        ], 
        "CWE": [
            {
                "url": "https://cwe.mitre.org/data/definitions/399.html", 
                "id": "CWE-399", 
                "title": "Resource Management Errors"
            }
        ], 
        "CPE": [
            {
                "id": "cpe:/o:canonical:ubuntu_linux:12.10"
            }, 
            {
                "id": "cpe:/a:python:python:3.3.0"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:13.04"
            }, 
            {
                "id": "cpe:/a:python:python:3.3.1"
            }, 
            {
                "id": "cpe:/a:python:python:3.3.2"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.0"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.1"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.2"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.3"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.4"
            }, 
            {
                "id": "cpe:/a:python:python:3.2.5"
            }, 
            {
                "id": "cpe:/o:canonical:ubuntu_linux:12.04:-:lts"
            }
        ]
    }, 
    "References": {
        "SCIP": [
            {
                "url": "http://www.scip.ch/?vuldb.65232", 
                "id": 65232
            }
        ], 
        "IAVM": [], 
        "BID": [], 
        "OSVDB": [], 
        "Other": {
            "References": [
                {
                    "url": "http://bugs.python.org/issue17980", 
                    "vendor": "CONFIRM"
                }, 
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=963260", 
                    "vendor": "MISC"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/USN-1985-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/USN-1984-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://www.ubuntu.com/usn/USN-1983-1", 
                    "vendor": "UBUNTU"
                }, 
                {
                    "url": "http://www.openwall.com/lists/oss-security/2013/05/16/6", 
                    "vendor": "MLIST"
                }, 
                {
                    "url": "http://secunia.com/advisories/55116", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://secunia.com/advisories/55107", 
                    "vendor": "SECUNIA"
                }, 
                {
                    "url": "http://rhn.redhat.com/errata/RHSA-2014-1690.html", 
                    "vendor": "REDHAT"
                }
            ]
        }, 
        "CertVN": []
    }, 
    "Risk": [
        {
            "CVSS v2": [
                {
                    "impact": "2.9", 
                    "access complexity": "medium", 
                    "confidentiality impact": "none", 
                    "authentication": "none", 
                    "exploit": "8.6", 
                    "base": "4.3", 
                    "integrity impact": "none", 
                    "availability impact": "partial", 
                    "access vector": "network"
                }
            ], 
            "severity": "Moderate", 
            "Top alert": false, 
            "Top vulnerable": false
        }
    ], 
    "Rules": {
        "Suricata": [], 
        "Snort": []
    }, 
    "Exploits": {
        "Metasploit": [], 
        "ExploitDB": [], 
        "Elliot D2": [], 
        "Saint": []
    }, 
    "vFeed": {
        "Product": "vFeed - The Correlated Vulnerability and Threat Database", 
        "Contact": "@toolswatch", 
        "Author": "NJ OUCHN", 
        "URL": "https://github.com/toolswatch/vFeed", 
        "Version": "0.6.5", 
        "id": "VFD-2013-2099"
    }, 
    "Scanners": {
        "Oval": [
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:18405", 
                "title": "USN-1983-1 -- python2.7 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:18405"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:18739", 
                "title": "USN-1984-1 -- python3.2 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:18739"
            }, 
            {
                "url": "http://oval.mitre.org/repository/data/getDef?id=oval:org.mitre.oval:def:19404", 
                "title": "USN-1985-1 -- python3.3 vulnerabilities", 
                "class": "patch", 
                "id": "oval:org.mitre.oval:def:19404"
            }
        ], 
        "Nmap": [], 
        "OpenVas": [
            {
                "name": "Fedora Update for zeroinstall-injector FEDORA-2013-12396", 
                "file": "gb_fedora_2013_12396_zeroinstall-injector_fc18.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866066"
            }, 
            {
                "name": "Fedora Update for zeroinstall-injector FEDORA-2013-12414", 
                "file": "gb_fedora_2013_12414_zeroinstall-injector_fc19.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866654"
            }, 
            {
                "name": "Fedora Update for zeroinstall-injector FEDORA-2013-12421", 
                "file": "gb_fedora_2013_12421_zeroinstall-injector_fc17.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866065"
            }, 
            {
                "name": "Fedora Update for python-pip FEDORA-2013-13140", 
                "file": "gb_fedora_2013_13140_python-pip_fc18.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866148"
            }, 
            {
                "name": "Fedora Update for python-pip FEDORA-2013-13213", 
                "file": "gb_fedora_2013_13213_python-pip_fc17.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866394"
            }, 
            {
                "name": "Fedora Update for python-pip FEDORA-2013-13216", 
                "file": "gb_fedora_2013_13216_python-pip_fc19.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "866701"
            }, 
            {
                "name": "Fedora Update for python3 FEDORA-2013-21415", 
                "file": "gb_fedora_2013_21415_python3_fc18.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "867086"
            }, 
            {
                "name": "Fedora Update for bzr FEDORA-2013-9620", 
                "file": "gb_fedora_2013_9620_bzr_fc18.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "865686"
            }, 
            {
                "name": "Fedora Update for bzr FEDORA-2013-9628", 
                "file": "gb_fedora_2013_9628_bzr_fc17.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "865700"
            }, 
            {
                "name": "Fedora Update for python-tornado FEDORA-2014-16477", 
                "file": "gb_fedora_2014_16477_python-tornado_fc19.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "867292"
            }, 
            {
                "name": "Ubuntu Update for python2.7 USN-1983-1", 
                "file": "gb_ubuntu_USN_1983_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "841589"
            }, 
            {
                "name": "Ubuntu Update for python3.2 USN-1984-1", 
                "file": "gb_ubuntu_USN_1984_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "841571"
            }, 
            {
                "name": "Ubuntu Update for python3.3 USN-1985-1", 
                "file": "gb_ubuntu_USN_1985_1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "841576"
            }
        ], 
        "Nessus": [
            {
                "name": "Amazon Linux AMI : python-tornado (ALAS-2015-521)", 
                "file": "ala_ALAS-2015-521.nasl", 
                "family": "Amazon Linux Local Security Checks", 
                "id": "83272"
            }, 
            {
                "name": "Fedora 18 : zeroinstall-injector-2.3-1.fc18 (2013-12396)", 
                "file": "fedora_2013-12396.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "68884"
            }, 
            {
                "name": "Fedora 19 : zeroinstall-injector-2.3-1.fc19 (2013-12414)", 
                "file": "fedora_2013-12414.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "68886"
            }, 
            {
                "name": "Fedora 17 : zeroinstall-injector-2.3-1.fc17 (2013-12421)", 
                "file": "fedora_2013-12421.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "68887"
            }, 
            {
                "name": "Fedora 18 : python-pip-1.3.1-4.fc18 (2013-13140)", 
                "file": "fedora_2013-13140.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "69062"
            }, 
            {
                "name": "Fedora 17 : python-pip-1.3.1-4.fc17 (2013-13213)", 
                "file": "fedora_2013-13213.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "69063"
            }, 
            {
                "name": "Fedora 19 : python-pip-1.3.1-4.fc19 (2013-13216)", 
                "file": "fedora_2013-13216.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "69064"
            }, 
            {
                "name": "Fedora 19 : bzr-2.5.1-11.fc19 (2013-9538)", 
                "file": "fedora_2013-9538.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "67366"
            }, 
            {
                "name": "Fedora 18 : bzr-2.5.1-11.fc18 (2013-9620)", 
                "file": "fedora_2013-9620.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "67369"
            }, 
            {
                "name": "Fedora 17 : bzr-2.5.1-11.fc17 (2013-9628)", 
                "file": "fedora_2013-9628.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "67371"
            }, 
            {
                "name": "Fedora 20 : python-tornado-2.2.1-7.fc20 (2014-16390)", 
                "file": "fedora_2014-16390.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "80061"
            }, 
            {
                "name": "Fedora 19 : python-tornado-2.2.1-7.fc19 (2014-16477)", 
                "file": "fedora_2014-16477.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "80134"
            }, 
            {
                "name": "Fedora 23 : python-pymongo-2.5.2-8.fc23 (2016-50abc3e885)", 
                "file": "fedora_2016-50abc3e885.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "89540"
            }, 
            {
                "name": "Fedora 22 : python-pymongo-2.5.2-8.fc22 (2016-52b294538d)", 
                "file": "fedora_2016-52b294538d.nasl", 
                "family": "Fedora Local Security Checks", 
                "id": "89544"
            }, 
            {
                "name": "GLSA-201401-04 : Python: Multiple vulnerabilities", 
                "file": "gentoo_GLSA-201401-04.nasl", 
                "family": "Gentoo Local Security Checks", 
                "id": "71811"
            }, 
            {
                "name": "Mandriva Linux Security Advisory : bzr (MDVSA-2013:229)", 
                "file": "mandriva_MDVSA-2013-229.nasl", 
                "family": "Mandriva Local Security Checks", 
                "id": "69842"
            }, 
            {
                "name": "openSUSE Security Update : python3 (openSUSE-SU-2014:1070-1)", 
                "file": "openSUSE-2014-517.nasl", 
                "family": "SuSE Local Security Checks", 
                "id": "77431"
            }, 
            {
                "name": "RHEL 6 : cloud-init (RHSA-2015:0042)", 
                "file": "redhat-RHSA-2015-0042.nasl", 
                "family": "Red Hat Local Security Checks", 
                "id": "80506"
            }, 
            {
                "name": "Ubuntu 12.04 LTS / 12.10 / 13.04 : python2.7 vulnerabilities (USN-1983-1)", 
                "file": "ubuntu_USN-1983-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "70267"
            }, 
            {
                "name": "Ubuntu 12.04 LTS / 12.10 : python3.2 vulnerabilities (USN-1984-1)", 
                "file": "ubuntu_USN-1984-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "70268"
            }, 
            {
                "name": "Ubuntu 12.10 / 13.04 : python3.3 vulnerabilities (USN-1985-1)", 
                "file": "ubuntu_USN-1985-1.nasl", 
                "family": "Ubuntu Local Security Checks", 
                "id": "70269"
            }
        ]
    }, 
    "Patches": {
        "Fedora": [], 
        "HP": [], 
        "Microsoft KB": [], 
        "Mandriva": [], 
        "Cisco": [], 
        "IBM AIX Apar": [], 
        "Suse": [], 
        "Redhat": [
            {
                "Redhat": {
                    "oval": "", 
                    "url": "https://rhn.redhat.com/errata/RHSA-2014-1690.html", 
                    "id": "RHSA-2014:1690", 
                    "title": ""
                }
            }
        ], 
        "Gentoo": [], 
        "Microsoft Bulletins": [], 
        "Ubuntu": [
            {
                "url": "http://www.ubuntu.com/usn/USN-1983-1", 
                "id": "USN-1983-1"
            }, 
            {
                "url": "http://www.ubuntu.com/usn/USN-1984-1", 
                "id": "USN-1984-1"
            }, 
            {
                "url": "http://www.ubuntu.com/usn/USN-1985-1", 
                "id": "USN-1985-1"
            }
        ], 
        "Debian": [], 
        "Vmware": []
    }
}